source/leuven/services/openondemand.rst

```diff
@@ -419,14 +419,20 @@ For more general information, please refer to the `official JupyterLab documenta
 RStudio Server
 --------------
 
-This interactive app allows you to run an RStudio session as a compute job.
-You will be running RStudio with R version 4.2.1.
-For more information on how to use RStudio, check out the `RStudio official documentation`_.
+This interactive app allows you to run an RStudio session on the cluster. 
+In the 'Toolchain year and R version' drop-down menu, you can control the R installation
+that will be used for this session (e.g. the `R/4.2.2-foss-2022b` module).
+Additionally, the R-bundle-CRAN and R-bundle-Bioconductor modules can be loaded 
+on top of the base R module to provide easy access to hundreds of preinstalled packages.
 
-The use is very similar to regular RStudio.
-It is recommended to install packages in a folder on your ``$VSC_DATA`` instead of the default location though,
-to avoid clogging your ``$VSC_HOME``.
-You can do this by using the ``lib`` argument for both the ``install.packages`` and the ``library`` function.
+It is also possible to use locally installed R packages with RStudio, see :ref:`R package management<r_package_management_standard_lib>`. 
+RStudio furthermore allows to create RStudio projects to manage your
+R environments. When doing so, we recommend to select the
```

Comment:
`R` and everywhere else perhaps?
We also have to double check this within our team perhaps, and across other pages. 
E.g. Python is not typeset as `Python`; so maybe the same also applies to R (or `R` ;-) )

Reply:
I think this change won't improve readability. Back-quotes should be reserved for code snippets and perhaps explicit module names. But to back quote every single instance of `R` is a bit over kill. We should also start doing this for `RStudio`, `MATLAB`, `Python` etc. I personally think leaving software names in text as they are and only put back-quotes around code that should jump out so it is easier to find back and perhaps copy by the user.